controller.go

```diff
@@ -648,6 +670,28 @@ func (c *Controller) syncHandler(key string) error {
 			}
 			if host.Status == provision.ActiveStatus {
 
+				if host.IP != "" {
```

Comment:
Is this needed specifically for Civo?

Reply:
No, this is present in all the other providers.